### ISSUE TITLE
Image-classification-transfer-learning-highlevel.ipynb still uses the sagemaker 1.x

### ISSUE THREAD

**Issue by sengstacken** [aws/amazon-sagemaker-examples#1879]
The notebook is still using sagemaker 1.x.  It won't run without downgrading to sagemaker 1.x.  The notebook needs to be updated to sagemaker 2.x since there were breaking changes between 1.x and 2.x

**Comment by ajaykarpur** [comment]
> It won't run

I was able to run this notebook end-to-end without any issues. There are some warning messages suggesting that the notebook should be upgraded to SDK v2, but this does not prevent you from running the notebook.

I have attached the `.ipynb` file demonstrating that the notebook runs without issues.

[Image-classification-transfer-learning-highlevel.ipynb.zip](https://github.com/aws/amazon-sagemaker-examples/files/5732538/Image-classification-transfer-learning-highlevel.ipynb.zip)
nstrating
